src/components/molecules/UiStepper/UiStepper.stories.ts

```diff
@@ -0,0 +1,449 @@
+import {
+  computed,
+  provide,
+  inject,
+} from 'vue';
+import type { WritableComputedRef } from 'vue';
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { useArgs } from '@storybook/preview-api';
+import { useArgTypes } from '@sb/helpers';
+import {
+  UiStepper,
+  UiProgress,
+  UiText,
+  UiList,
+} from '@index';
+import type { StepperProps } from '@index';
+import UiStepperStep from './_internal/UiStepperStep.vue';
+import type { StepperStepAttrsProps } from './_internal/UiStepperStep.vue';
+
+const stepperSteps: StepperStepAttrsProps[] = [
+  {
+    label: 'Introduction',
+    'data-testid': 'introduction',
+    href: '#',
+  },
+  {
+    label: 'A really long step label',
+    'data-testid': 'long-step-label',
+    href: '#',
+  },
+  {
+    label: 'Symptoms',
+    'data-testid': 'symptoms',
+    href: '#',
+  },
+  {
+    label: 'Regions',
+    'data-testid': 'regions',
+    href: '#',
+  },
+  {
+    label: 'Interview',
+    'data-testid': 'interview',
+    href: '#',
+  },
+  {
+    label: 'Results',
+    'data-testid': 'results',
+    href: '#',
+  },
+];
+
+type StepperArgsType = StepperProps;
+type StepperMetaType = Meta<StepperArgsType>;
+type StepperStoryType = StoryObj<StepperArgsType>;
+type StepperSteps = { steps: StepperStepAttrsProps[] };
+
+const { argTypes } = useArgTypes(UiStepper);
+const meta = {
+  title: 'Molecules/Stepper',
+  component: UiStepper,
+  args: {
+    currentStep: stepperSteps.at(3)?.label,
+    steps: stepperSteps,
+    progressAttrs: { id: 'progress' },
+  },
+  argTypes: {
+    ...argTypes,
+    currentStep: {
+      control: 'select',
+      options: stepperSteps.map((step) => step.label),
+    },
+    steps: { control: 'object' },
+    progressAttrs: {
+      table: { subcategory: 'Attrs props' },
+      control: 'object',
+    },
+  },
+  decorators: [ (story, { id }) => {
+    const [
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      { currentStep },
+      updateArgs,
+    ] = useArgs();
+    return {
+      components: { story },
+      setup(props, { attrs }) {
+        const currentStepAttrs = computed({
```

Comment:
I changed name of variable because eslint complained: ` ‘currentStep’ is already declared in the upper scope`